client-src/elements/chromedash-userlist.ts

```diff
@@ -93,21 +92,31 @@ class ChromedashUserlist extends LitElement {
   // TODO(jrobbins): Change this to be a JSON API call via csClient.
   async ajaxSubmit(e) {
     e.preventDefault();
-    const formEl = this.shadowRoot.querySelector('form');
+    const formEl: HTMLFormElement | null =
+      this.renderRoot.querySelector('form');
 
-    if (formEl.checkValidity()) {
-      const email = formEl.querySelector('input[name="email"]').value;
-      const isAdmin = formEl.querySelector('input[name="is_admin"]').checked;
-      const isSiteEditor = formEl.querySelector(
+    if (formEl && formEl.checkValidity()) {
```

Comment:
Instead of `x && x.y()`, use `x?.y()`. Except here, because the form is unconditionally in the rendered HTML, I'd just make the line above `...querySelector('form')!`.